Dockerfile

```diff
@@ -13,14 +13,16 @@ ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
 
-RUN pip install .
+RUN python setup.py install
+RUN pip install -r ./requirements.txt
```

Comment:
pls put a yum install gcc && pip install && yum erase gcc+deps somewhere here, so that it is in this layer and doesnt pollute the image ;) rm gcc on line 7